addons/website/views/snippets/s_image_text_overlap.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_image_text_overlap" name="Image - Text Overlap">
+    <section class="s_image_text_overlap pt64 pb64">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="9">
+                <div class="o_grid_item g-col-lg-6 g-height-7 col-lg-5 o_cc o_cc5 align-content-center pt16 pb16" style="grid-area: 2 / 7 / 9 / 13; --grid-item-padding-y: 48px; --grid-item-padding-x: 48px; z-index: 2; border-radius: var(--border-radius)">
+                    <h3>Services we offer</h3>
+                    <p>Write one or two paragraphs describing your product or services. To be successful your content needs to be useful to your readers.</p>
+                    <p>Start with the customer – find out what they want and give it to them.</p>
+                    <p><a href="#" class="btn btn-secondary">Learn more</a></p>
+                </div>
+                <div class="o_grid_item o_grid_item_image g-col-lg-7 g-height-9 col-lg-7 pt16 pb16" style="grid-area: 1 / 1 / 10 / 8; --grid-item-padding-x: 0px; z-index: 1;">
+                    <img src="/web/image/website.s_picture_default_image" class="img img-fluid mx-auto rounded" alt=""/>
+                </div>
```

Comment:
Website's conventions for grid items:

- Order element in the DOM according to the desktop visual result (top left to bottom right)
- Sequence  `z-index` values according to the DOM order  (⬆️ )
- If something needs to change position on mobile, add `order: [x];` inline style + the `order-lg-0` class